app/src/main/java/swati4star/createpdf/util/CreatePdf.java

```diff
@@ -164,6 +171,30 @@ protected String doInBackground(String... params) {
         return null;
     }
 
+    private void addPageNumber(Rectangle documentRect, PdfWriter writer) {
+        if (!mPageNumStyle.isEmpty()) {
+            ColumnText.showTextAligned(writer.getDirectContent(),
+                    Element.ALIGN_BOTTOM,
+                    getPhrase(writer, mPageNumStyle, mImagesUri.size()),
+                    ((documentRect.getRight() + documentRect.getLeft()) / 2),
+                    documentRect.getBottom() + 25, 0);
+        }
+    }
+
+    @NonNull
+    private Phrase getPhrase(PdfWriter writer, String pageNumStyle, int size) {
+        Phrase phrase;
+        if (pageNumStyle == Constants.PG_NUM_STYLE_PAGE_X_OF_N) {
+            phrase = new Phrase(String.format("Page %d of %d", writer.getPageNumber(), size));
```

Comment:
Take `Page` from strings.xml file.
So that when translations get added, the `Page ` here is also trnslated to given locale